test_autograph.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+## for testing autograph mar signing from signingscript
+
+mkdir -p artifact_dir/
+mkdir -p work_dir/cot/upstream-task-id1/
+
+test -f firefox-mozilla-central-58.0a1-linux-x86_64-en-US-20171001220301-20171002220204.partial.mar || (
+    wget http://ftp.mozilla.org/pub/firefox/nightly/latest-mozilla-central/firefox-mozilla-central-58.0a1-linux-x86_64-en-US-20171001220301-20171002220204.partial.mar \
+	&& \
+	cp firefox-mozilla-central-58.0a1-linux-x86_64-en-US-20171001220301-20171002220204.partial.mar work_dir/cot/upstream-task-id1/
+)
+test -f firefox-mozilla-central-58.0a1-linux-x86_64-en-US-20171001220301-20171002100134.partial.mar || (
+    wget http://ftp.mozilla.org/pub/firefox/nightly/latest-mozilla-central/firefox-mozilla-central-58.0a1-linux-x86_64-en-US-20171001220301-20171002100134.partial.mar \
```

Comment:
Nit: Like the previous comment, link should be https://archive.mozilla.org/pub/firefox/nightly/2017/10/2017-10-02-10-01-34-mozilla-central/firefox-mozilla-central-58.0a1-linux-x86_64-en-US-20171001220301-20171002100134.partial.mar